### PR TITLE
Initial version, closes #2, #4, #5, #6

### LINKED ISSUE #2: add driver client
Add driver client to be used by the server.

It should implement, at least:

```go
ParseUAST(req *protocol.ParseUASTRequest) (*protocol.ParseUASTResponse, error)
```

The rest of it should be something along the lines of:

```go
type Driver struct{}
func ExecDriver(r *runtime.Runtime, desc *DriverDescriptor) (*Driver, error)
func (d *Driver) ParseUAST(req *protocol.ParseUASTRequest) (*protocol.ParseUASTResponse, error)
func (d *Driver) Close() error
```

### PR DESCRIPTION
**Pull request by smola**

### CODE REVIEW

File: cmd/bblfsh/client.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	"github.com/bblfsh/server/runtime"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/bblfsh/sdk/protocol"
+	"google.golang.org/grpc"
+)
+
+type clientCmd struct {
+	Address     string `long:"address" description:"server address to connect to" default:"localhost:9432"`
+	Standalone  []bool `long:"standalone" description:"run standalone, without server"`
```
Comment (alcortesm) [inline cmd/bblfsh/client.go]: Should this be `bool` instead of `[]bool`?

File: driver.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	"io"
+
+	"github.com/bblfsh/server/runtime"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/bblfsh/sdk/protocol"
+	"github.com/bblfsh/sdk/protocol/driver"
+)
+
+type Driver interface {
```
Comment (alcortesm) [inline driver.go]: Document this interface.

File: cmd/bblfsh/client.go (hunk 1 above)
Comment (smola) [inline cmd/bblfsh/client.go]: Nope, go-flags mandates `[]bool`, which supports detecting things such as `-vvvvv`.

Comment (alcortesm) [inline cmd/bblfsh/client.go]: :+1:

File: cmd/bblfsh/client.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	RuntimePath string `long:"runtime-path" description:"runtime path for standalone mode" default:"/tmp/bblfsh-runtime"`
+	ImageRef    string `long:"image" value-name:"image-ref" description:"image reference to use (e.g. docker://bblfsh/python-driver:latest)"`
+	Language    string `long:"language" description:"language of the input" default:"auto"`
```
Comment (abeaumont) [inline cmd/bblfsh/client.go]: This option doesn't seem supported yet

**Pull request merged**